canonical-websites/snapcraft.io
# Update run script and dockerfile to use Bionic

Update run script and dockerfile to use Bionic as a base image, for dev and production.

## QA

`./run` - should function all as normal

Now try docker image:
- `docker build --build-arg COMMIT_ID=test --build-arg WEBAPP=snapcraft --tag test-sio .`
- `docker run --rm -p 80:80 -it -e SECRET_KEY=adsf test-sio`
